test/OgmiosDatumCache.purs

```diff
@@ -28,6 +32,23 @@ suite = group "Ogmios Datum Cache tests" $ do
     \https://github.com/mlabs-haskell/purescript-aeson/issues/7)"
     plutusDataToFromAesonTest
   test "Plutus data samples should have a compatible hash" plutusDataHashingTest
+  test "GetDatumsByHashesR fixture parses and hashes properly"
+    getDatumsByHashesHashingTest
+
+readGetDatumsByHashesSample
+  :: forall (m :: Type -> Type)
+   . MonadEffect m
+  => m GetDatumsByHashesR
+readGetDatumsByHashesSample = do
+  errEither <<< decodeAeson =<< readAeson
+    "./fixtures/test/ogmios-datum-cache/get-datums-by-hashes-samples.json"
+
+getDatumsByHashesHashingTest
+  :: forall m. Bind m => MonadEffect m => MonadThrow Error m => m Unit
```

Comment:
Odd that we have these effects in this style.

Anyway, m needs a kind annotation